pkg/ipam/node_manager.go

```diff
@@ -58,7 +58,7 @@ type NodeOperations interface {
 	// interfaces and IPs associated with the node. This function is called
 	// sparingly as this information is kept in sync based on the success
 	// of the functions AllocateIPs(), ReleaseIPs() and CreateInterface().
-	ResyncInterfacesAndIPs(ctx context.Context, scopedLog *logrus.Entry) (ipamTypes.AllocationMap, error)
+	ResyncInterfacesAndIPs(ctx context.Context, scopedLog *logrus.Entry) (allocatable ipamTypes.AllocationMap, remainingAvailableInterfaceCount int, err error)
```

Comment:
I think instead of having named returns here, could you document what the return values are meant to be in the godoc.

Reply:
it is done

Reply:
Could you remove the named returns in the interface declaration? There's no need for them there, just the godoc changes are fine.

Reply:
Ok, i remove the named returns